bot/bot.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from aiogram import Bot, Dispatcher, executor, types
+
+logging.basicConfig(level=logging.INFO)
+
+bot = Bot(token = '?')
```

Comment:
Change this to 
```python
bot = Bot(token=os.getenv('TOKEN')
```
It means you need to work this library [python-dotenv](https://pypi.org/project/python-dotenv/)
If you have problems with this, notice me in discussions